cmd/nodecmd/wiz.go

```diff
@@ -135,6 +136,7 @@ The node wiz command creates a devnet and deploys, sync and validate a subnet in
 	cmd.Flags().IntVar(&throughput, "aws-volume-throughput", constants.AWSGP3DefaultThroughput, "AWS throughput in MiB/s (for gp3 volume type only)")
 	cmd.Flags().StringVar(&volumeType, "aws-volume-type", "gp3", "AWS volume type")
 	cmd.Flags().IntVar(&volumeSize, "aws-volume-size", constants.CloudServerStorageSize, "AWS volume size in GB")
+	cmd.Flags().BoolVar(&cleanupNodes, "cleanup", false, "clean up all existing instances created by Avalanche CLI")
```

Comment:
I am not sure what it means in the scope of the `wiz`. delete all clusters registered  with avalanche-cli in case of `wiz` command failure?